### ISSUE TITLE
Integrate the app in a GPS app, e.g. OSMand

### ISSUE THREAD

**Issue by Gredin67** [covoiturage-libre/covoiturage-libre-android#2]
It would be awesome to be able to share carpooling information directly in a GPS navigation app such as OSMand. Is there any plan to develop an add-on for OSMand?

**Comment by Poussinou** [comment]
Not at all, but it could be a good idea. You mean something like the possibility to see the carpooling in the Google Maps app, too?

**Comment by Gredin67** [comment]
Well I think putting it on google maps would be against the idea of covoiturage-libre and free software. 

But yes, I mean 
- being able to see carpooling opportunities directly in OSMand (as a plugin for instance) 
- or maybe in a first step, send a ride planned in OSMand to covoiturage-libre-android app that will get or set a carpooling opportunit. If the journey is requested in pedestrian mode, then return cars on the same path. If in car mode, then publish a ride on covoiturage-libre.

For this to work properly, it would require OsmAnd to provide time scheduling in itinerary. Do you know how to bring interested people from OSM and covoiturage-libre together on github?

**Comment by Gredin67** [comment]
In order of importance : 

1. Automatically publish a carpooling ride on covoiturage-libre when a ride is planned in OsmAnd
2. Automatically open carpooling app and query for a ride when a pedestrian journey is planned in OsmAnd
